BaseTools/ImageTool/ElfScan.c

```diff
@@ -583,13 +585,18 @@ CreateIntermediate (
         memcpy (Segments[SIndex].Data, (UINT8 *)mEhdr + Shdr->sh_offset, (size_t)Shdr->sh_size);
       }
 
-      Segments[SIndex].ImageAddress = Pointer;
+      Segments[SIndex].ImageAddress = Shdr->sh_addr;
```

Comment:
Same here.

Reply:
Pointer is only needed here https://github.com/acidanthera/audk/blob/merge-min_size-assert/BaseTools/ImageTool/ElfScan.c#L620